nrf52-hal-common/src/uarte.rs

```diff
@@ -0,0 +1,153 @@
+//! HAL interface to the UARTE peripheral
+//!
+//! See product specification, chapter 33.
```

Comment:
Wrong chapter number. It's chapter 35 in the nRF52832 product specification, chapter 6.34 for nRF52840.

Reply:
Hm, I might have a stale version of the datasheet. It is Chapter 33 for me: https://send.firefox.com/download/46a875af6e/#CP1iWUg5VDe1gF2AawRvKg (OPS v0.6.3, which sounds suspiciously non-final).

Reply:
I have 1.4, which I just confirmed is the current one. I win :)